pages/reference/api/transcode-video-storj.en-US.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Transcode Video with Storj
+---
+
+## Introduction
+
+This guide will describe how to use the Livepeer [transcode API](/reference/api/transcode-video) to transcode and store video files in Storj using Storj’s S3 API.
+
+There are [code examples](#code-examples) provided for reference.
+
+## Storj S3 Credentials
+
+This step assumes that you have already created an account at [https://www.storj.io/](https://www.storj.io/).
+
+Follow the [Storj guide for generating S3 credentials](https://docs.storj.io/dcs/api-reference/s3-compatible-gateway#I7p7Q) using either the web interface or `uplink`. Make sure that the credentials have the proper read/write permissions for the bucket that your video files will be read from and that transcoded videos will be written to. The access key and secret key of the credentials will be used in the next step.
+
+## Upload Video to Storj
+
+Make sure you have a video file uploaded to your Storj bucket. You must upload the video file using one of the following methods:
+
+- Via the Storj web interface
+- Via a S3 API client
+    - [AWS has S3 API client implementations](https://aws.amazon.com/developer/tools/) in different programming languages
+    - The [code examples](#code-examples) illustrate how to upload a video file to Storj using a S3 API client
+
+Note: At the moment, you can *not* use `uplink` to upload the video file to Storj because `uplink` does not support generating the required S3 object metadata.
```

Comment:
Is this inconsistent with the comment below re: uplink? we say you can't use uplink for upload but then describe how you can use it for playback?